core/market/src/config.rs

```diff
@@ -33,37 +33,40 @@ pub struct SubscriptionConfig {
     pub default_ttl: chrono::Duration,
 }
 
-#[derive(Clone)]
+#[derive(StructOpt, Clone)]
 pub struct EventsConfig {
+    #[structopt(env = "MAX_MARKET_EVENTS_DEFAULT", default_value = "20")]
     pub max_events_default: i32,
+    #[structopt(env = "MAX_MARKET_EVENTS_MAX", default_value = "100")]
     pub max_events_max: i32,
 }
 
 impl Config {
     pub fn from_env() -> Result<Config, structopt::clap::Error> {
-        // Mock command line arguments, because we want to use ENV fallback
-        // or default values if ENV variables don't exist.
-        Ok(Config::from_iter_safe(vec!["yagna"].iter())?)
+        // Empty command line arguments, because we want to use ENV fallback
+        // or default values if ENV variables are not set.
+        Ok(Config::from_iter_safe(&[""])?)
```

Comment:
Does it work? First argument is expected to be binary name like in regular command line args

Reply:
yes, I've added test